client/Scenarios/ListByNameExpanded.cs

```diff
@@ -29,6 +30,38 @@ public override void Execute()
             {
                 Console.WriteLine($"--------VM id--------: {vm.Data.Id}");
             }
+            foreach (var networkInterface in rg.NetworkInterfaces().ListByName(Environment.UserName))
```

Comment:
Lets follow the style cop rules of having a space after the close brace to help with the habit for each of these for loops